Sources_v3/StreamChat/Workers/MessageUpdater.swift

```diff
@@ -353,3 +343,21 @@ private extension MessageDTO {
         localMessageState == .pendingSend || localMessageState == .sendingFailed
     }
 }
+
+private extension DatabaseSession {
+    func messageEditableByCurrentUser(_ messageId: MessageId) throws -> MessageDTO {
```

Comment:
Can you please add a comment explaining what is this good for?

Reply:
added in 093156304f98a746d9888f1478410046c0bf511e